src/main/java/htsjdk/variant/vcf/VCFCompoundHeaderLine.java

```diff
@@ -202,6 +202,9 @@ protected VCFCompoundHeaderLine(String line, VCFHeaderVersion version, Supported
     }
 
     private void validate() {
+        if (countType == VCFHeaderLineCount.INTEGER && count <= 0)
```

Comment:
The code style in this file is inconsistent, but it would be nice to migrate it to use `{` `}` for `if` blocks. Can you migrate the ones in this method as part of this PR?